PayPalDataCollector/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.paypal.android.paypaldatacollector">
+
+    <!-- following permission is optional -->
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
+    <!-- for reading GSF ID -->
```

Comment:
Are all these required? Looks like on BT we only use `INTERNET` and `ACCESS_NETWORK_STATE`

Reply:
i'll check, this is all on the magnes docs, but you maybe right about not needing all

Reply:
Yeah if we don't need access to fine grained location then that'll keep it from popping up on the install sheet in Google Play for all of our merchants.

Reply:
I agree with @sarahkoop the less permissions the better.